tests/integration/cri-containerd/gha-run.sh

```diff
@@ -35,7 +35,7 @@ function install_dependencies() {
 	sudo apt-get -y install "${system_deps[@]}"
 
 	ensure_yq
-	${repo_root_dir}/tests/install_go.sh -p -f
+	"${repo_root_dir}"/tests/install_go.sh -p -f
```

Comment:
You could enclose the path and the command using double quotes, and call it with the source command (which is an alias for `.` )
```suggestion
	. "${repo_root_dir}/tests/install_go.sh" -p -f
```